kolibri/plugins/coach_tools/assets/src/vue/reports/list-section/data-cells/progress-cell/index.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+
+  <div>
+    <progress-bar :progress="num"></progress-bar>
+    <div v-if="extraText" class="extra-text">
+      <svg src="./person.svg" class="person-icon"></svg>
+      {{extraText}}
+    </div>
+  </div>
+
+</template>
+
+
+<script>
+
+  module.exports = {
+    $trNameSpace: 'progress-indicator',
+    $trs: {
+      mastered: 'mastered by {0, number, integer} learners',
+      completed: 'completed by {0, number, integer} learners',
+      pct: '{0, number, percent}',
+    },
+    props: {
+      num: {
+        type: Number,
+        required: true,
+        validator(value) {
+          return value >= 0 && value <= 1;
+        },
+      },
+      isexercise: {
+        type: Boolean,
+        default: false,
+      },
+      numusers: {
+        type: Number,
+      },
+    },
+    computed: {
+      extraText() {
+        if (this.numusers === undefined) {
```

Comment:
The idea here was that it could actually show the text "mastered by 0 learners" if `0` was passed in, and distinguish `0` from `undefined`.

However I think I'll update this to have an explicit boolean property, `showNumUsers`.